### PR TITLE
feat(service): add service tech email support

### PR DESCRIPTION
**Pull request by rriski**
Add support for service tech emails

### CODE REVIEW

File: api/v1alpha1/common.go [hunk 1]
```diff
@@ -87,6 +88,10 @@ type ServiceCommonSpec struct {
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="Value is immutable"
 	// Service integrations to specify when creating a service. Not applied after initial service creation
 	ServiceIntegrations []*ServiceIntegrationItem `json:"serviceIntegrations,omitempty"`
+
+	// +kubebuilder:validation:MaxItems=10
+	// Technical contact emails of the service
+	TechnicalEmails *[]avnClient.ContactEmail `json:"tech_emails,omitempty"`
```
Comment (byashimov) [inline api/v1alpha1/common.go:94]: Hey. Please don't use client structs here. We need more control on that fields. Like adding validation rules.

**Review by byashimov — CHANGES_REQUESTED** [review]
Please add a test by extending any existing service test. Like Kafka, pg, etc.

**Comment by byashimov** [comment]
Btw please make sure that existing list can be purged. It is sufficient to test the behaviour on a local cluster, because the test might be complicated a bit.

File: api/v1alpha1/common.go (hunk 1 above)
Comment (rriski) [reply api/v1alpha1/common.go]: Added test in https://github.com/aiven/aiven-operator/pull/576/commits/f94a58472144580ed311e3cc6af0df71beb28689

**Comment by rriski** [comment]
> Btw please make sure that existing list can be purged. It is sufficient to test the behaviour on a local cluster, because the test might be complicated a bit.

Test case added in PR handles checking that emails can be deleted

**Review by byashimov — APPROVED** [review]
Great job.

**Pull request merged**